lib/routes/authorization.js

```diff
@@ -19,6 +18,13 @@ const verify = require('../browserid');
 const CODE = 'code';
 const TOKEN = 'token';
 
+const UNTRUSTED_CLIENT_SCOPES = [
```

Comment:
nit: the word-order in this name is confusing, it suggests the scopes themselves are somehow untrusted.  Something like `UNTRUSTED_CLIENT_ALLOWED_SCOPES` may be a little clearer.